h2o-algos/src/main/java/hex/rulefit/RuleFitModel.java

```diff
@@ -108,13 +108,15 @@ public RuleFitModel(Key<RuleFitModel> selfKey, RuleFitParameters parms, RuleFitO
 
     @Override
     public Frame score(Frame fr, String destination_key, Job j, boolean computeMetrics, CFuncRef customMetricFunc) throws IllegalArgumentException {
```

Comment:
Model#score is a public method that is not expected to run in the context of an existing Scope. it is called from REST API (ModelMetricsHandler).

You can only use Scope if you are also managing it, this code doesn't appear to do that.

Same issue on line 133.